cartridges/int_alma/cartridge/scripts/helpers/almaCheckoutHelper.js

```diff
@@ -58,10 +58,14 @@ function getPaymentCoreInfo(plan) {
  */
 function getCreditInfo(plan, currencyCode) {
     var costOfCredit = formatCurrency(plan.customer_total_cost_amount / 100, currencyCode);
+    var purchaseAmount = formatCurrency(plan.purchaseAmount, currencyCode);
+    var totalCost = formatCurrency(plan.purchaseAmount + (plan.customer_total_cost_amount / 100), currencyCode);
```

Comment:
1. `formatCurrency` is only available in `sfcc-cartridge-dev` repository... how does it works ? where `formatCurrency` come from here ? (`*/cartridge/scripts/util/formatting` not seem's to be a part of current repository)
2. I'am not sure that `formatMoney` or `Money` (used by `formatCurrency`) will round given amount. Can you confirm that for me please (or explain me why we don't use `Math.round()` here) ?